### ISSUE TITLE
Changes case when reading INI

### ISSUE THREAD

**Issue by spikespaz** [kellyjonbrazil/jc#285]
When running the following command:
```sh
cat MateriaDark_Original.kvconfig | jc --ini -p | bat
```
And inspecting the output, I can see
```json
{
    "%General": {
        "respect_de": "true",
    }
}
```

Whereas the original INI looks like:
```ini
[%General]
respect_DE=true
```

I see no option for preserving casing.

**Comment by kellyjonbrazil** [comment]
It looks like this is happening because I'm using the default behavior of the python `ConfigParser` library. Seems that it can be overridden with the `ConfigParser.optionxform(option)` attribute when constructing the `ConfigParser` instance.

I could change the behavior of `jc` such that the existing behavior stays in place by default (to preserve backwards compatibility) but preserve key-name case when `jc -r` is used.

**Comment by kellyjonbrazil** [comment]
I thought about this some more and I think it will be ok to make this breaking change to the default behavior. (I'll document it in the release notes)

I have updated the parser in the `dev` branch and this version will go into the next release in a few weeks. If you'd like to use it sooner, you can copy the parser file below to your plugin directory:
https://github.com/kellyjonbrazil/jc/blob/dev/jc/parsers/ini.py

Custom local parser plugins may be placed in a j`c/jcparsers` folder in your local "App data directory":

Linux/unix: `$HOME/.local/share/jc/jcparsers`
macOS: `$HOME/Library/Application Support/jc/jcparsers`
Windows: `$LOCALAPPDATA\jc\jc\jcparsers`

Note, this also affects the KeyValue (`--kv`) parser since they are one and the same.

**Comment by spikespaz** [comment]
Thank you!

**Comment by kellyjonbrazil** [comment]
merged into v1.22.0
